jekyll/_cci2/bitbucket-data-center-integration.adoc

```diff
@@ -0,0 +1,252 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Bitbucket Data Center integration overview
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Learn how to integrate Bitbucket Data Center with CircleCI to manage your pipelines with our.
+:icons: font
+:experimental:
+
+{% include snippets/bitbucket-data-center-closed-preview.adoc %}
+
+This page walks you through integrating a Bitbucket Data Center project with CircleCI. The sections below introduce you to concepts and ways to manage CI/CD (continuous integration and continuous delivery) pipelines for your Bitbucket Data Center project. CircleCI features that are in development for Bitbucket Data Center projects are detailed in the <<coming-soon>> section.
+
+[#overview]
+== CircleCI Bitbucket Data Center integration overview
+
+CircleCI CI/CD allows you to build, test (continuous integration) and deploy (continuous deployment, continuous delivery) your Bitbucket Data Center projects each time a developer pushes a change to the codebase, and/or when a merge request is created.
```

Comment:
So we won't run a build when a BBDC pull request is opened, but we will run a build once it's merged.